tekton/create-draft-triggers-release.yaml

```diff
@@ -0,0 +1,194 @@
+# Copyright 2019 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a modified version of the Task in the tektoncd/plumbing repo:
+# https://github.com/tektoncd/plumbing/blob/master/tekton/resources/release/github_release.yaml
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: create-draft-operator-release
```

Comment:
Bit confusion on file name and metadata.name because file name contains `triggers`  and metadata.name contains `operator` :thinking: